### DISCUSSION TITLE
Is Laravel Pulse out of beta now?

### DISCUSSION THREAD

**Discussion by mikenowakme** [laravel/framework#49328] [Q&A]
The Laravel docs still say to use the beta channel but I thought maybe since it was showing up on Github, it was out of beta now and we should switch to stable?

**Comment by antimech** [comment]
No, it is not. Beta version generally means the first version available to the public. You won't be able to install it without switching to the beta channel.
https://github.com/laravel/pulse/tags

**Comment by jbrooksuk** [comment]
Pulse is still in beta.
